extensions/jdbc/jdbc-oracle/jdbc-oracle-deployment/src/main/java/io/quarkus/jdbc/oracle/deployment/OracleProcessor.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.jdbc.oracle.deployment;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.substrate.RuntimeInitializedClassBuildItem;
+import io.quarkus.deployment.builditem.substrate.SubstrateResourceBundleBuildItem;
+
+class OracleProcessor {
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FeatureBuildItem.JDBC_ORACLE);
+    }
+
+    @BuildStep
+    SubstrateResourceBundleBuildItem includeResourceBundle() {
+        return new SubstrateResourceBundleBuildItem("oracle.net.jdbc.nl.mesg.NLSR");
+    }
+
+    @BuildStep
+    void initializeAtRuntime(BuildProducer<RuntimeInitializedClassBuildItem> initializeAtRuntime) {
+        initializeAtRuntime.produce(new RuntimeInitializedClassBuildItem("oracle.jdbc.driver.BlockSource"));
```

Comment:
I think best is to use
oracle.jdbc.driver.BlockSource.class instead of string.
Same for all other hardcoded string